onos-classic/values.yaml

```diff
@@ -54,3 +58,14 @@ ports:
 #     log4j2.appender.console.name = Console
 #     log4j2.appender.console.layout.type = PatternLayout
 #     log4j2.appender.console.layout.pattern = %d{RFC3339} %-5level [%c{1}] %msg%n%throwable
+
+podSecurityContext:
+  runAsUser: 1000
+  fsGroup: 2000
```

Comment:
Do we have any user with UID 1000 and group with GID 2000 in all images?

Reply:
I think this will be enforced by k8s